fenix/app/src/main/res/layout/full_screen_notification_dialog.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/full_screen_notification"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_margin="8dp"
+    android:background="@drawable/fenix_snackbar_background"
+    android:elevation="4dp"
+    android:minHeight="48dp"
+    android:orientation="horizontal"
+    android:paddingStart="16dp"
+    android:paddingEnd="16dp">
+
+    <TextView
+        android:id="@+id/full_screen_notification_text"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:ellipsize="end"
+        android:letterSpacing="0.05"
+        android:maxLines="2"
+        android:minHeight="46dp"
+        android:paddingTop="8dp"
+        android:paddingBottom="8dp"
+        android:text="@string/full_screen_notification"
+        android:textAlignment="textStart"
+        android:textColor="@color/photonWhite"
+        android:textSize="18sp"
```

Comment:
Sure! Do you mind providing a little more detail to ensure I cover your suggestion accurately in a ticket please

Reply:
Certainly, we're essentially duplicating https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/layout/fenix_snackbar.xml right now. Since the properties will be common across 2 different XML files, we want to be able to move the common properties into a `<style>` and be able to call it via `style="@style/SnackbarText"`. We may want to do this for any view that has a lot of duplication. This is to make sure we can change the snackbar style in a single source of truth. Otherwise, somebody would need to make the connection that this full screen notification dialog is also a snackbar which might not be obvious.

See https://searchfox.org/mozilla-mobile/source/firefox-android/fenix/app/src/main/res/values/styles.xml#402

Reply:
Cheers @gabrielluong, have created https://bugzilla.mozilla.org/show_bug.cgi?id=1863758